### PR TITLE
[docs] Updated 6lowpan docs to include Net/Dgram

### PR DESCRIPTION
**Pull request by jprestwo**
- OCF was only mentioned but the setup was the same
   for a 6lowpan device using Net/Dgram

Signed-off-by: James Prestwood <james.prestwood@intel.com>

### CODE REVIEW

File: docs/6lowpan-ble.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# OCF over BLE
-This document describes the steps to build and run an OCF server over BLE on
-the Arduino 101. This setup requires a Linux desktop (or similar) with a BLE
-adapter and 6LoWPAN support. The 6LoWPAN module is standard on Ubuntu and
-should work with a 4.0 compliant BT/BLE adapter.
+# Using 6LoWPAN
+This document describes the steps to build and run an ZJS IP application that
```
Comment (grgustaf) [inline docs/6lowpan-ble.md]: a ZJS

File: docs/6lowpan-ble.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+uses 6LoWPAN as its IP transport.This setup requires a Linux desktop
```
Comment (grgustaf) [inline docs/6lowpan-ble.md]: space after period

File: docs/6lowpan-ble.md [hunk 3]
```diff
@@ -14,23 +14,29 @@ should work with a 4.0 compliant BT/BLE adapter.
...
+$ make BOARD=arduino_101 JS=samples/TCPEchoServ6.js DEVICE_NAME="my_tcp_server"
 ```
+When doing BLE discovery (explained below) you should see your device name and
+MAC address appear if you use these options.
 
-## Flashing the A101
-Flash the Arduino 101 as you would any other script:
-```bash
-$ make dfu
-```
```
Comment (grgustaf) [inline docs/6lowpan-ble.md]: Did you really mean to remove the dfu part?

Comment (jprestwo) [inline docs/6lowpan-ble.md]: I figured the user should know how to flash, this doc was really just meant to describe the BLE connection procedure and different make variables. I can put it back in if you want though

Comment (grgustaf) [inline docs/6lowpan-ble.md]: Fair enough, just checking.

**Comment by grgustaf** [comment]
+1

**Pull request merged**